### DISCUSSION TITLE
button to set thermostat in a preset mode for a time

### DISCUSSION THREAD

**Discussion by GitHubMe78** [jmcollin78/versatile_thermostat#573] [General]
Hi JM
Is it possible to add a button to the card to set boost mode for say 15 mn and come back to the previous state afterwards ?
Merci d'avance

**Comment by jmcollin78** [comment]
Hello,

There was already some feature request like this in the past: https://github.com/jmcollin78/versatile_thermostat/issues/59
But it have been refused because no real need to integrate it into the core feature of VTherm (this is a specific need).

You can always propose a PR if you want or simulate it with an automation of your side
